Templates/UnifiedMultiplayerGem/Template/gem.json

```
@@ -21,5 +22,9 @@
     "documentation_url": "${DocumentationURL}",
     "dependencies": [
```

I think this gem should declare a dependency on Multiplayer because it is used in the CMakeList.txt
```suggestion
    "dependencies": [
        "Multiplayer"
```